code/modules/xenoarcheology/effect.dm

```diff
@@ -12,20 +10,11 @@
 	var/artifact_id = ""
 	var/effect_type = 0
 
-//0 = Unknown / none detectable
-//1 = Concentrated energy
-//2 = Intermittent psionic wavefront
-//3 = Electromagnetic energy
-//4 = Particle field
-//5 = Organically reactive exotic particles
-//6 = Interdimensional/bluespace? phasing
-//7 = Atomic synthesis
-
```

Comment:
Did you move these comments or did you just delete them. If you deleted them can you put them back. If you move them, can explain where you moved them?  If they are not required anymore, can explain why they are not required anymore?